pkg/operation/cloudbotanist/vagrantbotanist/miscellaneous.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 The Gardener Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vagrantbotanist
+
+// ApplyCreateHook does currently nothing for Vagrant.
+func (b *VagrantBotanist) ApplyCreateHook() error {
+	return nil
+}
+
+// ApplyDeleteHook does currently nothing for Vagrant.
+func (b *VagrantBotanist) ApplyDeleteHook() error {
+	return nil
+}
+
+// CheckIfClusterGetsScaled does currently nothing for Vagrant, as ScaleSets aren't supported.
+func (b *VagrantBotanist) CheckIfClusterGetsScaled() (bool, int, error) {
```

Comment:
Function is not longer part of the cloud botanist interface, please remove it.

Reply:
removed